examples/cc26xx/cc26xx-web-demo/cc26xx-web-demo.c

```diff
@@ -966,6 +996,56 @@ PROCESS_THREAD(cc26xx_web_demo_process, ev, data)
 
   PROCESS_END();
 }
+
+PROCESS_THREAD(adc_process, ev, data)
+{
+  PROCESS_BEGIN();
+  static struct etimer et_adc;
+  while(1) {
+    etimer_set(&et_adc, CLOCK_SECOND * 5);
```

Comment:
That line should be moved above the "`while(1) {`" line.  It doesn't need to be executed more than once.